src/User.js

```diff
@@ -1,14 +1,47 @@
 class User {
-  constructor(user){
+  constructor(user) {
     this.id = user.id;
     this.name = user.name;
-    this.customer = user.customer;
     this.userBookings = [];
+    this.bookingCost = 0;
```

Comment:
Is this "total" booking cost? If so, would `totalBookingCost` be easier to understand and remember? To me, `bookingCost` suggests the cost of a single booking.